README.md

```diff
@@ -5,6 +5,39 @@
 This python script helps you build a full [GTK](https://www.gtk.org/) library
 stack for Windows using Visual Studio. Currently, GTK 3 and GTK 4 are supported.
 
+## Install GTK Only
+
+If you want to only run GTK on Windows and not build it yourself, you can download
+a zip file from the last release and unzip it to `C:\gtk`.
+
+It comes with GTK4, Cairo, PyGObject, Pycairo, GtkSourceView5, adwaita-icon-theme, and
+all of their dependencies.
+
+Unfortunately, these binaries are provided as-is, and we can't guarantee timely updates

```

Comment:
I think we need a stronger statement, eg:

Note however that these binaries are provided “AS IS”, WITHOUT WARRANTY OF ANY KIND.
They just contain the output of our latest CI run. They are not tested and we cannot commit to timely updates even for security issues. We strongly recommend to build your own binaries, especially if you plan to distribute them with your application or use them in production.

Reply:
Hi @pbor, I updated the statement to match your wording.